content/_data/changelogs/weekly.yml

```diff
@@ -7411,6 +7411,112 @@
   # pull: 4674 (PR title: Fix rendering of links in the Maintainer guide)
   # pull: 4678 (PR title: Reduce code duplication in SystemProperties)
 
+- version: '2.235'
+  date: 2020-05-04
+  changes:
+  - type: major bug
+    category: regression
+    pull: 4701
+    issue: 62133
+    authors:
+    - daniel-beck
+    message: |-
+      When the resource root URL configuration pointed at a previously set up resource root URL, form validation showed a "404 Not Found" error. (regression in 2.205).
```

Comment:
Wrong form.

Reply:
Thanks for the feedback!

I'm not sure what you meant by "wrong form".  I've reworded in 571bb17d64e19b0736b6f3ebc274772a15eb0d12 to match the phrasing and punctuation of a few previous entries that mention '(regression in xxx)'.  Let me know if that's not what you were trying to help me see.